service/internal/access/v2/evaluate_test.go

```diff
@@ -751,11 +799,16 @@ func (s *EvaluateTestSuite) TestEvaluateResourceAttributeValues() {
 
 // Test cases for getResourceDecision
 func (s *EvaluateTestSuite) TestGetResourceDecision() {
+	levelRegResValueFQN := createAttrValueFQN(baseNamespace, "level", "registered_res")
```

Comment:
are these attribute value  FQNs or registered resource value FQNs?

Reply:
updated to use `createRegisteredResourceValueFQN` and different var names